bootstrap.yml

```diff
@@ -62,6 +62,16 @@
         shell: podman pull quay.io/quads/badfish
         ignore_errors: yes
 
+      - name: get SELinux status
+        shell: getenforce
+        register: selinux_status
+
+      - set_fact: se_enforcing=false
```

Comment:
I don't think a separate fact is required here. Just check directly for `when: '"Enforcing" in selinux_status.stdout'`
in line 75 in `post.yml`.